modules/smart-agent_redis/conf/04-stored-keys.yaml

```diff
@@ -0,0 +1,15 @@
+module: redis
+name: stored keys change rate
+
+filtering: "filter('plugin', 'redis_info')"
+
```

Comment:
```suggestion
disable: true
tip: no change on keyspace over a long period can indicate it is full. if you don't use redis as cache but as queue broker or database so it can be normal to not see any activity depending on your application and you should disable this detector

```